internal/test/e2e/cluster.go

```diff
@@ -731,55 +726,11 @@ func (c *TestCluster) WaitForContracts() []api.ContractMetadata {
 	c.waitForHostContracts(hostsMap)
 
 	// fetch all contracts
-	resp, err := c.Bus.Contracts(context.Background(), api.ContractsOpts{
-		FilterMode: api.ContractFilterModeActive,
-	})
+	resp, err := c.Bus.Contracts(context.Background(), api.ContractsOpts{})
```

Comment:
Since we no longer have `WaitForContractSet`, this should probably filter for good contracts to get the same functionality.